HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -244,6 +244,7 @@ def initialize(hpxml_path: nil, collapse_enclosure: true)
     # Clean up
     delete_partition_surfaces()
     delete_tiny_surfaces()
+    delete_adiabatic_subsurfaces()
```

Comment:
I ended up implementing adiabatic door deletion inside hpxml class scope. In this way the net area of its attached wall won't be messed up. I have run the other housing unit test file for confirmation.

Reply:
This is clever, but I worry that it might cause [these error checks](https://github.com/NREL/OpenStudio-HPXML/blob/d26e36c45415d8057c819f45319b3ef9a2bd0504/HPXMLtoOpenStudio/measure.rb#L310-L374) to get triggered? Maybe you can call hpxml.delete_adiabatic_subsurfaces() from the end of the linked method instead of in the HPXML constructor?

Reply:
In the error checking, I didn't see it do anything with subsurfaces. The deletion only applied to subsurfaces, or more specifically, doors, along with its attachment to its parent surface. Since it didn't delete parent surface, the error-cheking should be fine. Also, I noticed that we also created hpxml class in our [test](https://github.com/NREL/OpenStudio-HPXML/blob/d26e36c45415d8057c819f45319b3ef9a2bd0504/workflow/tests/hpxml_translator_test.rb#L509), if we only implemented deletion in the measure, we might need to make changes to test codes as well to address it. So implementing in class constructor might be simpler.

Reply:
Oops! Disregard what I said. I previously read the method "delete_adiabatic_subsurfaces" as "delete_adiabatic_**surfaces**".